templates/ExprEval.cpp

```diff
@@ -39,8 +39,7 @@
 #include <sstream>
 #include <string_view>
 
-using namespace UHDM;
-
```

Comment:
Most changes in this file are irrelevant to the problem at hand.
Do not perform these changes in this PR.